### ISSUE TITLE
Installation issue on Linux Mint

### ISSUE THREAD

**Issue by marcomilanesio** [pymc-devs/pymc#1373]
Hi,

Installation does not work on Linux Mint.
Thanks

```
[11:46:27] $ sudo pip3 install -U git+https://github.com/pymc-devs/pymc3.git
The directory '/home/marco/.cache/pip/http' or its parent directory is not owned by the current user and the cache has been disabled. Please check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
The directory '/home/marco/.cache/pip' or its parent directory is not owned by the current user and caching wheels has been disabled. check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
Collecting git+https://github.com/pymc-devs/pymc3.git
  Cloning https://github.com/pymc-devs/pymc3.git to /tmp/pip-zwb65y0s-build
Requirement already up-to-date: numpy>=1.11.0 in /usr/local/lib/python3.4/dist-packages (from pymc3==3.0rc1)
Collecting scipy>=0.12.0 (from pymc3==3.0rc1)
  Downloading scipy-0.18.0-cp34-cp34m-manylinux1_x86_64.whl (40.3MB)
    100% |████████████████████████████████| 40.3MB 43kB/s 
Requirement already up-to-date: matplotlib>=1.5.0 in /usr/local/lib/python3.4/dist-packages (from pymc3==3.0rc1)
Requirement already up-to-date: theano>=0.8.2 in /usr/local/lib/python3.4/dist-packages (from pymc3==3.0rc1)
Requirement already up-to-date: pandas>=0.18.0 in /usr/local/lib/python3.4/dist-packages (from pymc3==3.0rc1)
Requirement already up-to-date: patsy>=0.4.0 in /usr/local/lib/python3.4/dist-packages (from pymc3==3.0rc1)
Requirement already up-to-date: joblib>=0.9 in /usr/local/lib/python3.4/dist-packages (from pymc3==3.0rc1)
Requirement already up-to-date: CommonMark==0.5.4 in /usr/local/lib/python3.4/dist-packages (from pymc3==3.0rc1)
Requirement already up-to-date: recommonmark in /usr/local/lib/python3.4/dist-packages (from pymc3==3.0rc1)
Collecting sphinx (from pymc3==3.0rc1)
  Downloading Sphinx-1.4.6-py2.py3-none-any.whl (1.6MB)
    100% |████████████████████████████████| 1.6MB 738kB/s 
Requirement already up-to-date: nbsphinx in /usr/local/lib/python3.4/dist-packages (from pymc3==3.0rc1)
Requirement already up-to-date: numpydoc in /usr/local/lib/python3.4/dist-packages (from pymc3==3.0rc1)
Requirement already up-to-date: python-dateutil in /usr/local/lib/python3.4/dist-packages (from matplotlib>=1.5.0->pymc3==3.0rc1)
Requirement already up-to-date: pytz in /usr/local/lib/python3.4/dist-packages (from matplotlib>=1.5.0->pymc3==3.0rc1)
Requirement already up-to-date: cycler in /usr/local/lib/python3.4/dist-packages (from matplotlib>=1.5.0->pymc3==3.0rc1)
Requirement already up-to-date: pyparsing!=2.0.4,!=2.1.2,>=1.5.6 in /usr/local/lib/python3.4/dist-packages (from matplotlib>=1.5.0->pymc3==3.0rc1)
Requirement already up-to-date: six>=1.9.0 in /usr/local/lib/python3.4/dist-packages (from theano>=0.8.2->pymc3==3.0rc1)
Requirement already up-to-date: docutils>=0.11 in /usr/local/lib/python3.4/dist-packages (from recommonmark->pymc3==3.0rc1)
Collecting babel!=2.0,>=1.3 (from sphinx->pymc3==3.0rc1)
  Downloading Babel-2.3.4-py2.py3-none-any.whl (7.1MB)
    100% |████████████████████████████████| 7.1MB 235kB/s 
Collecting Jinja2>=2.3 (from sphinx->pymc3==3.0rc1)
  Downloading Jinja2-2.8-py2.py3-none-any.whl (263kB)
    100% |████████████████████████████████| 266kB 8.0MB/s 
Collecting alabaster<0.8,>=0.7 (from sphinx->pymc3==3.0rc1)
  Downloading alabaster-0.7.9-py2.py3-none-any.whl
Collecting imagesize (from sphinx->pymc3==3.0rc1)
  Downloading imagesize-0.7.1-py2.py3-none-any.whl
Collecting snowballstemmer>=1.1 (from sphinx->pymc3==3.0rc1)
  Downloading snowballstemmer-1.2.1-py2.py3-none-any.whl (64kB)
    100% |████████████████████████████████| 71kB 2.1MB/s 
Requirement already up-to-date: Pygments>=2.0 in /usr/local/lib/python3.4/dist-packages (from sphinx->pymc3==3.0rc1)
Requirement already up-to-date: nbconvert in /usr/local/lib/python3.4/dist-packages/nbconvert-4.2.0-py3.4.egg (from nbsphinx->pymc3==3.0rc1)
Requirement already up-to-date: nbformat in /usr/local/lib/python3.4/dist-packages/nbformat-4.1.0-py3.4.egg (from nbsphinx->pymc3==3.0rc1)
Collecting MarkupSafe (from Jinja2>=2.3->sphinx->pymc3==3.0rc1)
  Downloading MarkupSafe-0.23.tar.gz
Collecting mistune!=0.6 (from nbconvert->nbsphinx->pymc3==3.0rc1)
  Downloading mistune-0.7.3-py2.py3-none-any.whl
Requirement already up-to-date: traitlets in /usr/local/lib/python3.4/dist-packages/traitlets-4.3.0-py3.4.egg (from nbconvert->nbsphinx->pymc3==3.0rc1)
Requirement already up-to-date: jupyter_core in /usr/local/lib/python3.4/dist-packages/jupyter_core-4.2.0-py3.4.egg (from nbconvert->nbsphinx->pymc3==3.0rc1)
Collecting entrypoints (from nbconvert->nbsphinx->pymc3==3.0rc1)
  Downloading entrypoints-0.2.2-py2.py3-none-any.whl
Requirement already up-to-date: ipython_genutils in /usr/local/lib/python3.4/dist-packages/ipython_genutils-0.1.0-py3.4.egg (from nbformat->nbsphinx->pymc3==3.0rc1)
Requirement already up-to-date: jsonschema!=2.5.0,>=2.0 in /usr/local/lib/python3.4/dist-packages (from nbformat->nbsphinx->pymc3==3.0rc1)
Collecting decorator (from traitlets->nbconvert->nbsphinx->pymc3==3.0rc1)
  Downloading decorator-4.0.10-py2.py3-none-any.whl
Installing collected packages: scipy, babel, MarkupSafe, Jinja2, alabaster, imagesize, snowballstemmer, sphinx, pymc3, mistune, entrypoints, decorator
  Found existing installation: scipy 0.13.3
    DEPRECATION: Uninstalling a distutils installed project (scipy) has been deprecated and will be removed in a future version. This is due to the fact that uninstalling a distutils project will only partially uninstall the project.
    Uninstalling scipy-0.13.3:
      Successfully uninstalled scipy-0.13.3
  Found existing installation: Babel 2.2.0
    Uninstalling Babel-2.2.0:
      Successfully uninstalled Babel-2.2.0
  Found existing installation: MarkupSafe 0.18
    Uninstalling MarkupSafe-0.18:
      Successfully uninstalled MarkupSafe-0.18
  Running setup.py install for MarkupSafe ... done
  Found existing installation: Jinja2 2.7.2
    Uninstalling Jinja2-2.7.2:
      Successfully uninstalled Jinja2-2.7.2
  Found existing installation: Sphinx 1.2.3
    Uninstalling Sphinx-1.2.3:
      Successfully uninstalled Sphinx-1.2.3
  Found existing installation: pymc3 3.0rc1
    Uninstalling pymc3-3.0rc1:
      Successfully uninstalled pymc3-3.0rc1
  Running setup.py install for pymc3 ... error
    Complete output from command /usr/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-zwb65y0s-build/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-470w92gw-record/install-record.txt --single-version-externally-managed --compile:
    WARNING: '.' not a valid package name; please use only .-separated package names in setup.py
    /usr/local/lib/python3.4/dist-packages/setuptools-20.2.2-py3.4.egg/setuptools/dist.py:285: UserWarning: Normalizing '3.0.rc1' to '3.0rc1'
    running install
    running build
    running build_py
    creating build
    creating build/lib
    creating build/lib/pymc3
    copying pymc3/interactive_sampling.py -> build/lib/pymc3
    copying pymc3/plots.py -> build/lib/pymc3
    copying pymc3/sampling.py -> build/lib/pymc3
    copying pymc3/progressbar.py -> build/lib/pymc3
    copying pymc3/vartypes.py -> build/lib/pymc3
    copying pymc3/model.py -> build/lib/pymc3
    copying pymc3/data.py -> build/lib/pymc3
    copying pymc3/__init__.py -> build/lib/pymc3
    copying pymc3/debug.py -> build/lib/pymc3
    copying pymc3/theanof.py -> build/lib/pymc3
    copying pymc3/math.py -> build/lib/pymc3
    copying pymc3/diagnostics.py -> build/lib/pymc3
    copying pymc3/memoize.py -> build/lib/pymc3
    copying pymc3/stats.py -> build/lib/pymc3
    copying pymc3/blocking.py -> build/lib/pymc3
    creating build/lib/pymc3/distributions
    copying pymc3/distributions/multivariate.py -> build/lib/pymc3/distributions
    copying pymc3/distributions/__init__.py -> build/lib/pymc3/distributions
    copying pymc3/distributions/distribution.py -> build/lib/pymc3/distributions
    copying pymc3/distributions/dist_math.py -> build/lib/pymc3/distributions
    copying pymc3/distributions/discrete.py -> build/lib/pymc3/distributions
    copying pymc3/distributions/timeseries.py -> build/lib/pymc3/distributions
    copying pymc3/distributions/special.py -> build/lib/pymc3/distributions
    copying pymc3/distributions/transforms.py -> build/lib/pymc3/distributions
    copying pymc3/distributions/continuous.py -> build/lib/pymc3/distributions
    creating build/lib/pymc3/step_methods
    copying pymc3/step_methods/ATMCMC.py -> build/lib/pymc3/step_methods
    copying pymc3/step_methods/gibbs.py -> build/lib/pymc3/step_methods
    copying pymc3/step_methods/quadpotential.py -> build/lib/pymc3/step_methods
    copying pymc3/step_methods/__init__.py -> build/lib/pymc3/step_methods
    copying pymc3/step_methods/nuts.py -> build/lib/pymc3/step_methods
    copying pymc3/step_methods/metropolis.py -> build/lib/pymc3/step_methods
    copying pymc3/step_methods/compound.py -> build/lib/pymc3/step_methods
    copying pymc3/step_methods/arraystep.py -> build/lib/pymc3/step_methods
    copying pymc3/step_methods/hmc.py -> build/lib/pymc3/step_methods
    copying pymc3/step_methods/slicer.py -> build/lib/pymc3/step_methods
    creating build/lib/pymc3/tuning
    copying pymc3/tuning/__init__.py -> build/lib/pymc3/tuning
    copying pymc3/tuning/scaling.py -> build/lib/pymc3/tuning
    copying pymc3/tuning/starting.py -> build/lib/pymc3/tuning
    creating build/lib/pymc3/tests
    copying pymc3/tests/test_missing.py -> build/lib/pymc3/tests
    copying pymc3/tests/test_transforms.py -> build/lib/pymc3/tests
    copying pymc3/tests/test_examples.py -> build/lib/pymc3/tests
    copying pymc3/tests/test_text_backend.py -> build/lib/pymc3/tests
    copying pymc3/tests/test_model_func.py -> build/lib/pymc3/tests
    copying pymc3/tests/test_special_functions.py -> build/lib/pymc3/tests
    copying pymc3/tests/test_tracetab.py -> build/lib/pymc3/tests
    copying pymc3/tests/test_distribution_defaults.py -> build/lib/pymc3/tests
    copying pymc3/tests/backend_fixtures.py -> build/lib/pymc3/tests
    copying pymc3/tests/test_diagnostics.py -> build/lib/pymc3/tests
    copying pymc3/tests/__init__.py -> build/lib/pymc3/tests
    copying pymc3/tests/knownfailure.py -> build/lib/pymc3/tests
    copying pymc3/tests/test_ndarray_backend.py -> build/lib/pymc3/tests
    copying pymc3/tests/test_memo.py -> build/lib/pymc3/tests
    copying pymc3/tests/test_shared.py -> build/lib/pymc3/tests
    copying pymc3/tests/helpers.py -> build/lib/pymc3/tests
    copying pymc3/tests/test_sampling.py -> build/lib/pymc3/tests
    copying pymc3/tests/test_step.py -> build/lib/pymc3/tests
    copying pymc3/tests/test_plots.py -> build/lib/pymc3/tests
    copying pymc3/tests/test_distributions.py -> build/lib/pymc3/tests
    copying pymc3/tests/test_hmc.py -> build/lib/pymc3/tests
    copying pymc3/tests/test_stats.py -> build/lib/pymc3/tests
    copying pymc3/tests/test_tuning.py -> build/lib/pymc3/tests
    copying pymc3/tests/test_posdef_sym.py -> build/lib/pymc3/tests
    copying pymc3/tests/test_distributions_random.py -> build/lib/pymc3/tests
    copying pymc3/tests/test_glm.py -> build/lib/pymc3/tests
    copying pymc3/tests/test_advi.py -> build/lib/pymc3/tests
    copying pymc3/tests/test_profile.py -> build/lib/pymc3/tests
    copying pymc3/tests/checks.py -> build/lib/pymc3/tests
    copying pymc3/tests/test_starting.py -> build/lib/pymc3/tests
    copying pymc3/tests/test_sqlite_backend.py -> build/lib/pymc3/tests
    copying pymc3/tests/models.py -> build/lib/pymc3/tests
    creating build/lib/pymc3/glm
    copying pymc3/glm/__init__.py -> build/lib/pymc3/glm
    copying pymc3/glm/families.py -> build/lib/pymc3/glm
    copying pymc3/glm/glm.py -> build/lib/pymc3/glm
    creating build/lib/pymc3/examples
    copying pymc3/examples/best.py -> build/lib/pymc3/examples
    copying pymc3/examples/lasso_block_update.py -> build/lib/pymc3/examples
    copying pymc3/examples/latent_occupancy.py -> build/lib/pymc3/examples
    copying pymc3/examples/factor_potential.py -> build/lib/pymc3/examples
    copying pymc3/examples/disaster_model_arbitrary_deterministic.py -> build/lib/pymc3/examples
    copying pymc3/examples/garch_example.py -> build/lib/pymc3/examples
    copying pymc3/examples/dirichlet.py -> build/lib/pymc3/examples
    copying pymc3/examples/hierarchical.py -> build/lib/pymc3/examples
    copying pymc3/examples/disaster_model.py -> build/lib/pymc3/examples
    copying pymc3/examples/simpletest.py -> build/lib/pymc3/examples
    copying pymc3/examples/wishart.py -> build/lib/pymc3/examples
    copying pymc3/examples/__init__.py -> build/lib/pymc3/examples
    copying pymc3/examples/ATMIP_2gaussians.py -> build/lib/pymc3/examples
    copying pymc3/examples/rsv.py -> build/lib/pymc3/examples
    copying pymc3/examples/logistic.py -> build/lib/pymc3/examples
    copying pymc3/examples/hierarchical_sqlite.py -> build/lib/pymc3/examples
    copying pymc3/examples/discrete_find_MAP.py -> build/lib/pymc3/examples
    copying pymc3/examples/speedtest_pymc3.py -> build/lib/pymc3/examples
    copying pymc3/examples/disaster_model_missing.py -> build/lib/pymc3/examples
    copying pymc3/examples/glm_linear.py -> build/lib/pymc3/examples
    copying pymc3/examples/python_vs_c.py -> build/lib/pymc3/examples
    copying pymc3/examples/lightspeed_example.py -> build/lib/pymc3/examples
    copying pymc3/examples/arbitrary_stochastic.py -> build/lib/pymc3/examples
    copying pymc3/examples/ARM12_6.py -> build/lib/pymc3/examples
    copying pymc3/examples/ARM12_6uranium.py -> build/lib/pymc3/examples
    copying pymc3/examples/lasso_missing.py -> build/lib/pymc3/examples
    copying pymc3/examples/custom_dists.py -> build/lib/pymc3/examples
    copying pymc3/examples/GHME_2013.py -> build/lib/pymc3/examples
    copying pymc3/examples/tutorial.py -> build/lib/pymc3/examples
    copying pymc3/examples/LKJ_correlation.py -> build/lib/pymc3/examples
    copying pymc3/examples/ARM5_4.py -> build/lib/pymc3/examples
    copying pymc3/examples/gelman_schools.py -> build/lib/pymc3/examples
    copying pymc3/examples/glm_robust.py -> build/lib/pymc3/examples
    copying pymc3/examples/banana.py -> build/lib/pymc3/examples
    copying pymc3/examples/nutstest.py -> build/lib/pymc3/examples
    copying pymc3/examples/arma_example.py -> build/lib/pymc3/examples
    copying pymc3/examples/gelman_bioassay.py -> build/lib/pymc3/examples
    copying pymc3/examples/stochastic_volatility.py -> build/lib/pymc3/examples
    copying pymc3/examples/normal.py -> build/lib/pymc3/examples
    creating build/lib/pymc3/backends
    copying pymc3/backends/sqlite.py -> build/lib/pymc3/backends
    copying pymc3/backends/tracetab.py -> build/lib/pymc3/backends
    copying pymc3/backends/text.py -> build/lib/pymc3/backends
    copying pymc3/backends/__init__.py -> build/lib/pymc3/backends
    copying pymc3/backends/base.py -> build/lib/pymc3/backends
    copying pymc3/backends/ndarray.py -> build/lib/pymc3/backends
    creating build/lib/pymc3/variational
    copying pymc3/variational/advi.py -> build/lib/pymc3/variational
    copying pymc3/variational/__init__.py -> build/lib/pymc3/variational
    copying pymc3/variational/advi_minibatch.py -> build/lib/pymc3/variational
    package init file 'docs/__init__.py' not found (or not a regular file)
    package init file '__init__.py' not found (or not a regular file)
    copying setupegg.py -> build/lib
    copying setup.py -> build/lib
    creating build/lib/pymc3/examples/data
    copying pymc3/examples/data/pancreatitis.csv -> build/lib/pymc3/examples/data
    copying pymc3/examples/data/jester-dataset-v1-dense-first-1000.csv -> build/lib/pymc3/examples/data
    copying pymc3/examples/data/HtWt.csv -> build/lib/pymc3/examples/data
    copying pymc3/examples/data/__init__.py -> build/lib/pymc3/examples/data
    copying pymc3/examples/data/wells.dat -> build/lib/pymc3/examples/data
    copying pymc3/examples/data/Guber1999data.txt -> build/lib/pymc3/examples/data
    copying pymc3/examples/data/srrs2.dat -> build/lib/pymc3/examples/data
    copying pymc3/examples/data/cty.dat -> build/lib/pymc3/examples/data
    copying pymc3/examples/data/test_scores.csv -> build/lib/pymc3/examples/data
    copying pymc3/examples/data/SP500.csv -> build/lib/pymc3/examples/data
    copying pymc3/examples/data/radon.csv -> build/lib/pymc3/examples/data
    creating build/lib/docs
    creating build/lib/docs/source
    creating build/lib/docs/source/notebooks
    copying docs/source/notebooks/advi.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/gaussian_mixture_model.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/profiling.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/GLM-logistic.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/lda-advi-aevb.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/dp_mix.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/discrete_find_MAP.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/getting_started.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/rolling_regression.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/GLM.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/howto_debugging.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/GLM-robust.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/Model Comparison.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/GLM-model-selection.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/disaster_model.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/GLM-robust-with-outlier-detection.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/posterior_predictive.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/gaussian-mixture-model-advi.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/pmf-pymc.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/stochastic_volatility.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/tutorial.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/GP-smoothing.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/GLM-hierarchical-advi-minibatch.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/GLM-poisson-regression.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/cox_model.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/GLM-hierarchical.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/lasso_block_update.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/rugby_analytics.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/BEST.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/survival_analysis.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/bayesian_neural_network_advi.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/GLM-linear.ipynb -> build/lib/docs/source/notebooks
    copying docs/source/notebooks/GLM-hierarchical-ADVI.ipynb -> build/lib/docs/source/notebooks
    creating build/lib/docs/source/data
    copying docs/source/data/pancreatitis.csv -> build/lib/docs/source/data
    copying docs/source/data/jester-dataset-v1-dense-first-1000.csv -> build/lib/docs/source/data
    copying docs/source/data/HtWt.csv -> build/lib/docs/source/data
    copying docs/source/data/__init__.py -> build/lib/docs/source/data
    copying docs/source/data/wells.dat -> build/lib/docs/source/data
    copying docs/source/data/Guber1999data.txt -> build/lib/docs/source/data
    error: can't copy 'docs/source/data/pmf-map-d5': doesn't exist or not a regular file

    ----------------------------------------
  Rolling back uninstall of pymc3
Command "/usr/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-zwb65y0s-build/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-470w92gw-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-zwb65y0s-build/

```

**Comment by twiecki** [comment]
Thanks for reporting, related to #1370. Not sure what packaging problem that is yet but it's high priority.

**Comment by fonnesbeck** [comment]
I don't see where those data subdirectories get used anywhere, so the expedient fix would be to ditch them.

**Comment by fonnesbeck** [comment]
Scratch that; I see where the data are used now.

**Comment by springcoil** [comment]
Shall we close this once this is merged in :)

**Comment by fonnesbeck** [comment]
The PR will close it automatically when merged.

**Comment by aloctavodia** [comment]
I just run into the same problem. I am using Ubuntu and Anaconda. Running
`pip install -U --force-reinstall git+https://github.com/pymc-devs/pymc3.git` does not solve the problem neither installing sphinx trough Anaconda as mention on #1370. I also tried installing PyMC3 inside a new environments, but no luck.

**Comment by twiecki** [comment]
I attempted to fix here: https://github.com/pymc-devs/pymc3/pull/1399 can you try if that helps?

**Comment by twiecki** [comment]
`pip install -U --force-reinstall git+https://github.com/pymc-devs/pymc3.git@manifest`

Can you give this a try @aloctavodia?

**Comment by aloctavodia** [comment]
upps, sorry for the delay. This fix the problem, Thanks!

**Comment by twiecki** [comment]
Thanks @aloctavodia. Closed by #1399
